src/cdvdcontrol.cpp

```diff
@@ -0,0 +1,1433 @@
+    std::cout << "\t--as-del #                   AS: delete strategy #\n";
+    std::cout << "\t--as-create <q|f> <a|r>      AS: create new strategy for inserted DVD media (PX-755/PX-760 only)\n";
+    std::cout << "\t                             mode:    q = quick, f = full\n";
+    std::cout << "\t                             action:  a = add,   r = replace\n";
+    std::cout << "\t--as-save <file>             AS: save Strategies            / EXPERIMENTAL /\n";
+    std::cout << "\t--as-load <file>             AS: load Strategy from file    / EXPERIMENTAL /\n";
+    std::cout << "\t--as-clear                   AS: remove ALL strategies from database\n";
+    std::cout << "\t--dvd+testwrite [on|off]     turn on/off testwrite on DVD+R(W) media\n";
+    std::cout << "\t--silent [on|off]            just turn Silent Mode on/off (default: on)\n";
+    std::cout << "\t--sm-nosave                  don't save Silent Mode settings\n";
+    std::cout << "\t--sm-cd-rd #                 set max CD READ speed (default: 32X)\n";
+    std::cout << "\t                             values: 4, 8, 24, 32, 40, 48\n";
+    std::cout << "\t--sm-cd-wr #                 set max CD WRITE speed (default: 32X)\n";
+    std::cout << "\t                             values: 4, 8, 16, 24, 32, 48\n";
+    std::cout << "\t--sm-dvd-rd #                set max DVD READ speed (default: 12X)\n";
+    std::cout << "\t                             values: 2, 5, 8, 12, 16\n";
+//  std::cout << "\t--sm-dvd-wr #                set max DVD WRITE speed (default: 8X)\n";
+//  std::cout << "\t                             values: 4, 6, 8, 12, 16\n";
+    std::cout << "\t--sm-load #                  set tray load speed. spd can be 0 to 80 (default: 63)\n";
+    std::cout << "\t--sm-eject #                 set tray eject speed. spd can be 0 to 80 (default: 0)\n";
+    std::cout << "\t--sm-access <slow|fast>      set access time, has effect only with CD/DVD speed setting\n";
+    std::cout << "\t--destruct <quick|full>      perform PlexEraser function\n";
+    std::cout << "\t                             WARNING! data on inserted CD-R/DVD-R will be lost!\n";
+    std::cout << "\t                             works on PX-755/PX-760/Premium2 only\n";
+
+    std::cout << "\noptions for Pioneer devices:\n";
+    std::cout << "\t--pio-limit [on|off]         limit (or not) read speed by 24x for CD and 8x for DVD\n";
+    std::cout << "\t--pio-quiet [quiet|perf|std] select QuietMode setting: Quiet, Performance or Standard (default: quiet)\n";
+    std::cout << "\t--pio-nosave                 don't save settings to drive (changes will be lost after reboot)\n";
+}
+
+void needs_parameter(char* argv[], int i) {
+    std::cout << "option " << argv[i] << " needs parameter!\n";
+}
+
+auto main (int argc, char* argv[]) -> int
+{
+    const int eleven_bit_limit = 2048;
+    const int six_bit_limit_minus_one = 63;
+    const int eight_bit_limit = 256;
+    int i=0;
+    int drvcnt=0;
```

Comment:
Codacy found an issue: [The scope of the variable 'drvcnt' can be reduced.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)